client/dashboard/plugins/scheduled-updates/index.tsx

```diff
@@ -1,19 +1,141 @@
+import { hostingUpdateSchedulesQuery, sitesQuery } from '@automattic/api-queries';
+import { useQuery } from '@tanstack/react-query';
 import { useNavigate } from '@tanstack/react-router';
-import { Button } from '@wordpress/components';
+import { Button, FormToggle } from '@wordpress/components';
+import { DataViews, type Field, filterSortAndPaginate, View } from '@wordpress/dataviews';
 import { __ } from '@wordpress/i18n';
+import { format, fromUnixTime } from 'date-fns';
+import { useState, useMemo } from 'react';
 import {
 	pluginsScheduledUpdatesNewRoute,
 	pluginsScheduledUpdatesRoute,
 } from '../../app/router/plugins';
+import { DataViewsCard } from '../../components/dataviews-card';
 import { PageHeader } from '../../components/page-header';
 import PageLayout from '../../components/page-layout';
+import { SiteIconLink } from '../../sites/site-fields';
+import { ScheduledUpdateRow } from './types';
+
+export const fields: Field< ScheduledUpdateRow >[] = [
+	{
+		id: 'site',
+		type: 'text',
+		label: __( 'Site' ),
+		getValue: ( { item } ) => item.site.name,
+	},
+	{
+		id: 'lastUpdate',
+		type: 'text',
+		label: __( 'Last Update' ),
+		render: ( { item } ) =>
+			item.lastUpdate ? format( fromUnixTime( item.lastUpdate ), 'MMM d, yyyy H:mm' ) : '-',
+	},
+	{
+		id: 'nextUpdate',
+		type: 'text',
+		label: __( 'Next Update' ),
+		render: ( { item } ) => format( fromUnixTime( item.nextUpdate ), 'MMM d, yyyy H:mm' ),
+	},
```

Comment:
Sorting doesn't work for these fields in my testing. I tested with three schedules on the same day, but with different hours.